src/main/java/com/atlauncher/App.java

```diff
@@ -1017,6 +1021,11 @@ private static void parseCommandLineArguments(String[] args) {
             System.exit(0);
         }
 
+        if (options.has("version")) {
+            System.out.println(Constants.VERSION);
```

Comment:
Can you change this to print out `System.out.println(String.format("%s %s", Constants.LAUNCHER_NAME, Constants.VERSION.toStringForLogging()));`